src/session/content/message_row/bubble.rs

```diff
@@ -284,15 +284,18 @@ impl MessageBubble {
             imp.sender_label.remove_css_class(&old_class);
         }
 
-        let color_class =
-            SENDER_COLOR_CLASSES[sender_id.map(|id| id as usize).unwrap_or_else(|| {
-                let mut s = DefaultHasher::new();
-                imp.sender_label.label().hash(&mut s);
-                s.finish() as usize
-            }) % SENDER_COLOR_CLASSES.len()];
-
-        imp.sender_label.add_css_class(color_class);
-        imp.sender_color_class.replace(Some(color_class.into()));
+        //Add sender css class only if message is incoming
+        if !self.has_css_class("outgoing") {
```

Comment:
It's much better checking for the actual outgoing property instead of a general css class.